ContestModel/src/org/icpc/tools/contest/model/IResolveInfo.java

```diff
@@ -32,6 +32,13 @@ public String toString() {
 	 */
 	double getSpeedFactor();
 
+	/**
+	 * Returns the scroll speed factor. Normal speed is 1, <1 is faster and >1 is slower.
```

Comment:
In `TeamIntroPresentation`, `timeFactor` > 1 is faster and <1 is slower. There, timeFactor 2 is double the speed. Here, speedFactor .5 is half the time.

Reply:
it's actually the opposite here as well :) >1 is faster, <1 is slower

Reply:
Fixed.